scripts/linux/github/updateLatestVersion.sh

```diff
@@ -243,15 +243,20 @@ update_latest_version_json()
 #######################################
 github_update_and_push()
 {
-    [[ -z "$AZURE_IOTEDGE_REPO_PATH" ]] && { echo "\$IOTEDGE_REPO_PATH is undefined"; exit 1; }
+    [[ -z "$AZURE_IOTEDGE_REPO_PATH" ]] && { echo "\$AZURE_IOTEDGE_REPO_PATH is undefined"; exit 1; }
     [[ -z "$VERSION" ]] && { echo "\$VERSION is undefined"; exit 1; }
     [[ -z "$GITHUB_PAT" ]] && { echo "\$GITHUB_PAT is undefined"; exit 1; }
 
     cd $AZURE_IOTEDGE_REPO_PATH
     git config user.name iotedge1
+
+    git checkout main
```

Comment:
Does this command work from a pipeline? I tried it once and recall getting an error stating that I wasn't in a git repo.

Reply:
Depending on which image of 1ES you are using. For Ubuntu18.04 & Ubuntu20.04, it should work with our current build images.